<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGameWindow</class>
 <widget class="QDialog" name="AddGameWindow">
  <property name="windowTitle">
   <string>Добавить игру</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выберите игру:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="existingGameComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="copiesLabel">
       <property name="text">
        <string>Количество копий:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="copiesSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="newGameGroupBox">
     <property name="title">
      <string>Создать новую игру</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="titleLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="publisherLabel">
        <property name="text">
         <string>Издатель:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="publisherLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="yearLabel">
        <property name="text">
         <string>Год выпуска:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="yearSpinBox">
        <property name="minimum">
         <number>1900</number>
        </property>
        <property name="maximum">
         <number>2100</number>
        </property>
        <property name="value">
         <number>2023</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="genresLabel">
        <property name="text">
         <string>Жанры:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QListWidget" name="genresListWidget"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Описание:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QTextEdit" name="descriptionTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addGameButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
